### PR TITLE
ACS 7.2.0 and 7.1.1 (start of code freeze for ACS 7.2.0-M2 and 7.1.1 GA)

### PR DESCRIPTION
**Pull request by alandavis**
* Repo: 7.2.0-A20 and 7.1.1-A5
* Share: 7.2.0-A20 and 7.1.1-A5
* T-Router: 1.5.0-A3
* T-Engine: 2.5.5-A3
* SFS: 0.16.1-A1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ The table below shows the exact version of ACS deployed with each chart version/
 | 5.1.0             | 7.1.0.1  |         | 7.0.1 | 6.2.2 | 6.1.1 | 7.1.0     |
 | 5.1.1             | 7.1.0.1  |         | 7.0.1 | 6.2.2 | 6.1.1 | 7.1.0     |
 | 5.2.0-M1          | 7.2.0-M1 | 7.1.0.1 | 7.0.1 | 6.2.2 | 6.1.1 | 7.2.0-M1  |
+| 5.2.0-?           | 7.2.0-A20 | 7.1.1-A5 | 7.0.1 | 6.2.2 | 6.1.1 | 7.2.0-M1  |
```
Comment (alxgomz) [inline README.md:39]: Isn't it M2?

**Review by alxgomz — CHANGES_REQUESTED** [review]

**Comment by alxgomz** [comment]
You'll also need to update the Chart.yml before merging otherwise you'll endup overwriting the 5.2.0-M1 incubator release

**Review by alxgomz — CHANGES_REQUESTED** [review]
Make sure to bump chart version in `acs-deployment/helm/alfresco-content-services/Chart.yaml` otherwise you'll endup overwriting 5.2.0-M1 in incubator

**Comment by alandavis** [comment]
Changed the version as requested https://github.com/Alfresco/acs-deployment/pull/641#pullrequestreview-858551059 but why am I having to do this? The devops team should have released the 5.2.0-M1 from the incubator to stable weeks ago at the end of the last train's code freeze. I note that the sync-service has already been updated on master for the current code freeze, so 5.2.0-M1 already has been overwritten. I will leave the devops team to fix that.

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request closed without merging**